.github/workflows/workflow.yaml

```diff
@@ -50,13 +53,35 @@ jobs:
           	EOF
           fi
 
+      - name: Run tests (bzlmod)
+        if: matrix.bazel_mode == 'module'
+        working-directory: ./tests
+        run: |
           echo "USE_BAZEL_VERSION=${{ matrix.version }}" > .bazeliskrc
 
-      - name: Build & Test
-        run: |
+          # Make the configs also available in the tests directory.
+          cp ../.bazelrc.local .
+          cp ../.bazelrc .
+
           if [[ ${{ runner.os }} == Windows ]]; then
             # On Windows `//...` expands to `/...`.
-            BAZEL_SH='C:\msys64\usr\bin\bash.exe' bazel test ///...
+            BAZEL_SH='C:\msys64\usr\bin\bash.exe' bazel test  ///...
+          else
+            bazel test  //...
+          fi
+
+      - name: Run tests (workspace)
+        if: matrix.bazel_mode == 'workspace'
+        working-directory: ./tests
+        run: |
+          echo "USE_BAZEL_VERSION=${{ matrix.version }}" > .bazeliskrc
+
+          # Make the configs also available in the tests directory.
+          cp ../.bazelrc.local .
+          cp ../.bazelrc .
```

Comment:
`tests/` could use a permanent `.bazelrc` so that it is also correctly configured in local development. To that end you can add `tests/.bazelrc` which contains `import %workspace%/../.bazelrc` and check it in.
You'll still need to copy `.bazelrc.local` and it will be covered by the `try-import %workspace%/.bazelrc.local` line in `.bazelrc` in the repository root.

Reply:
I've also included `import %workspace%/../.bazelrc.local` so there won't be a need for copying.